DataFormats/simulation/include/SimulationDataFormat/DigitizationContext.h

```diff
@@ -85,6 +90,11 @@ class DigitizationContext
   /// returns the GRP object associated to this context
   o2::parameters::GRPObject const& getGRP() const;
 
+  // helper functions to save and load a context
+  void saveToFile(const char* filename) const;
```

Comment:
not very important, but in most cases we use "const std::string& name" or ``const std::string_view name`` instead of raw pointers, perhaps we should do this everywhere?

Reply:
`std::string_view`, please. `std::string` will allocate a temporary object.

Reply:
changed to std::string_view